pkg/controller/awselb/aws_elb_controller.go

```diff
@@ -56,10 +60,16 @@ const (
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	maxRetries     = 15
 	controllerName = "awselb"
+
+	// elbResyncDuration represents period of time after which we will re-add machines to the ELBs.
+	// Once successfully added we add a timestamp to status, each time the machine is synced we check if
+	// this duration is exceeded and if so, re-add to the ELB to ensure we repair any machines
+	// accidentally taken out of rotation.
+	elbResyncDuration = 2 * time.Hour
```

Comment:
Although not necessary now, eventually we should add command-line arguments to allow this to be configurable.